.github/insert_printable_hyperlinks.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+import fileinput
+import pathlib
+import re
+
+HYPERLINKS_TO_REPLACE = [
+    "AIrules",
+    "Ability",
+    "Combat",
+    "Defend",
+    "Difficulty",
+    "End",
+    "Endcombat",
+    "Level",
+    "Movement",
+    "Playerdecks",
+    "Quick",
+    "Trading",
+    "Walls",
+]
+SECTIONS_DIRECTORY = "sections"
```

Comment:
Use Path instead of strings from the start
```suggestion
SECTIONS_DIRECTORY = pathlib.Path(__file__).parent.parent / "sections"
```